Set up codecov action

It's basically ready, but I need to configure the pelkmans lab org first to activate the Codecov app there

Now I get:

```
error - 2025-01-09 11:19:53,770 -- Upload failed: {"message":"Token required because branch is protected"}
==> Failed to upload coverage
    Exiting...
Error: Process completed with exit code 1.
```

--- a/.github/workflows/build_and_test.yml
+++ b/.github/workflows/build_and_test.yml
@@ -61,12 +61,12 @@ jobs:
       - name: Test core library with pytest
         run: python -m pytest --cov=src --cov-report=xml --cov-report=term tests 
 
-      # - name: Upload coverage to Codecov
-      #   uses: codecov/codecov-action@v5
-      #   with:
-      #     files: ./coverage.xml
-      #     fail_ci_if_error: true
-      #     token: ${{ secrets.CODECOV_TOKEN }}
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v5
+        with:
+          files: ./coverage.xml
+          fail_ci_if_error: true
+          token: ${{ secrets.CODECOV_TOKEN }}
 
   deploy:
     name: Deploy

--- a/README.md
+++ b/README.md
@@ -9,9 +9,23 @@
 
 | Task | Description | Passing |
 | --- | --- | --- |
-| Convert Cellvoyager Multiplexing to existing OME-Zarr | Converts CV7000/CV8000 images and extends to existing OME-Zarr file.|x|
-| Calculate ITK Elastix Registration | Compute rigid/affine/b-spline registration for aligning images in multiplexed image analysis.|x|
+| Convert Cellvoyager Multiplexing to existing OME-Zarr | Converts CV7000/CV8000 images and extends to existing OME-Zarr file.| WIP |
+| Compute Registration (elastix) | Compute rigid/affine/b-spline registration for aligning images in multiplexed image analysis.|x|
+| Apply Registration (elastix) | Apply rigid/affine/b-spline registration to images.|x|
 
 ## Installation
 
-TBD
+To install this task package on a Fractal server, get the whl in the Github release and use the local task collection.
+To install this package locally:
+```
+git clone https://github.com/pelkmanslab/abbott
+cd abbott
+pip install -e .
+```
+
+For development:
+```
+git clone https://github.com/pelkmanslab/abbott
+cd abbott
+pip install -e ".[dev]"
+```

--- a/README.md
+++ b/README.md
@@ -1,4 +1,7 @@
 # abbott
+[![License](https://img.shields.io/badge/License-BSD_3--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+[![CI Status](https://github.com/pelkmanslab/abbott/actions/workflows/build_and_test.yml/badge.svg)](https://github.com/pelkmanslab/abbott/actions/workflows/build_and_test.yml)
+[![codecov](https://codecov.io/github/pelkmanslab/abbott/graph/badge.svg?token=BF9NP4YLO6)](https://codecov.io/github/pelkmanslab/abbott)
 
 3D Multiplexed Image Analysis Workflows
 

